Shelfie.Identity/Shelfie.Identity.Api/Controllers/AuthenticateController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using Shelfie.Identity.Api.Models;
+using Shelfie.Identity.BusinessLogic.UseCases.LoginUser;
+using Shelfie.Identity.BusinessLogic.UseCases.RegisterUser;
+
+namespace Shelfie.Identity.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class AuthenticateController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public AuthenticateController(
+        IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    [HttpPost]
+    [Route("login")]
+    public async Task<IActionResult> Login([FromBody] LoginModel model)
+    {
+        var request = new LoginUserCommand
+        {
+            Username = model.Username,
+            Password = model.Password
+        };
+
+        var result = await _mediator.Send(request);
+
+        if (result is null)
```

Comment:
1. You do not have consistency with brackets - in some ifs you use the and in some not
2. Reconsider returned status. Unauthorized is thrown but it is method without any authorization needed. If result is null probably wrong username or password is inserted